GPU/Common/TextureCacheCommon.cpp

```diff
@@ -314,8 +295,7 @@ void TextureCacheCommon::UpdateMaxSeenV(TexCacheEntry *entry, bool throughMode)
 	}
 }
 
-
-void TextureCacheCommon::SetTexture(bool force) {
+TexCacheEntry *TextureCacheCommon::SetTexture(bool force) {
 #ifdef DEBUG_TEXTURES
 	if (SetDebugTexture()) {
```

Comment:
I know it's ifdef'd out for debugging, but might be nice to fix this - though I'm not sure how exactly it should return an entry, maybe a fake one or null, I guess?

-[Unknown]

Reply:
I'm not even entirely sure what the purpose of this thing is?

Reply:
Hm, it's been there a long time.  It's actually probably not useful at all now that we have the GE debugger... we should just remove.  It was basically to highlight a different texture every 30 frames, in a cycle, so you could narrow down which texture was being cached wrong.

-[Unknown]